### PR TITLE
React Router

### PR DESCRIPTION
**Pull request by Behnazz**
I made three Routes for /Events, /Admin, Event/eventname.
The eventname should be change later to be dynamic.

I also used Boostrap for Forms to make them look nicer :)

### CODE REVIEW

File: frontend/src/index.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import "bootstrap/dist/css/bootstrap.css";
```
Comment (dahfool) [inline frontend/src/index.js]: Bootstrap has been added in 2 places, here and in the index.html.

File: frontend/public/index.html [hunk 2]
```diff
@@ -19,14 +22,15 @@
...
     -->
-    <title>React App</title>
-  </head>
-  <body>
-    <noscript>
-      You need to enable JavaScript to run this app.
-    </noscript>
-    <div id="root"></div>
-    <!--
+  <title>React App</title>
```
Comment (dahfool) [inline frontend/public/index.html]: Update title to Events Calendar

File: frontend/src/Components/Events.js [hunk 3]
```diff
@@ -1,11 +1,10 @@
 import React from "react";
-// import { testData } from "../TestData.js";
-import { Event } from "./Event.js";
+import Event from "./Event.js";
 
 const Events = props => {
     return (
         <div className="events">
```
Comment (dahfool) [inline frontend/src/Components/Events.js:6]: From the list of events, you should be able to click on an event to find more info about that event.

File: frontend/src/index.js (hunk 1 above)
Comment (Behnazz) [reply frontend/src/index.js]: I removed it from index.js

File: frontend/src/Components/Admin-form.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+export default class Form extends React.Component {
+    state = {
+        EventsName: '',
+        EventsDate: '',
+        Description: ' '
```
Comment (alismith-dev) [inline frontend/src/Components/Admin-form.js]: It looks like your description defaults to a space inside the string. I would change that to an empty string.

File: frontend/src/Components/Events.js (hunk 3 above)
Comment (alismith-dev) [reply frontend/src/Components/Events.js:6]: I agree with @dahfool, but I think we can do that as future work. For now I think we can move ahead with this.

**Review by 40thieves — APPROVED** [review]
LGTM :shipit:

**Comment by alismith-dev** [comment]
I'm going to merge this, as I think @dahfool's comment can be addressed in future work and all the other review comments have been addressed

**Pull request merged**